### PR TITLE
Remove unused SubmissionIntroFragment (EXPOSUREAPP-4187)

### PR DESCRIPTION
**Pull request by Oliver-Zimmerman**
### Description

Removed SubmissionInroFragment, its layout, view model and module as well as relevant tests. 

### Steps to reproduce
N/A

### CODE REVIEW

**Review by ralfgehrer — DISMISSED** [review]

File: Corona-Warn-App/src/main/res/layout/fragment_submission_intro.xml [hunk 1]
```diff
@@ -1,58 +0,0 @@
...
-        <include
-            android:id="@+id/submission_intro_header"
-            layout="@layout/include_header"
-            android:layout_width="@dimen/match_constraint"
-            android:layout_height="wrap_content"
-            app:icon="@{@drawable/ic_close}"
-            app:layout_constraintEnd_toEndOf="parent"
-            app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toTopOf="parent"
-            app:title="@{@string/submission_intro_title}" />
```
Comment (chiljamgossow) [inline Corona-Warn-App/src/main/res/layout/fragment_submission_intro.xml:24]: Those strings could be removed as well

Comment (Oliver-Zimmerman) [reply Corona-Warn-App/src/main/res/layout/fragment_submission_intro.xml:24]: Done

Comment (ritsam) [reply Corona-Warn-App/src/main/res/layout/fragment_submission_intro.xml:24]: Got an Error while building locally 
_fragment_submission_test_result_available.xml:83: error: resource string/submission_intro_button_next_

**Review by ralfgehrer — APPROVED** [review]

**Review by AndroidMedaGalaxy — APPROVED** [review]
Tested and LGTM  (ﾉ◕ヮ◕)ﾉ*:・ﾟ✧

**Pull request merged**